client/components/edit/ObjectStatus.tsx

```diff
@@ -1,13 +1,28 @@
-import styles from "./ObjectStatus.module.css";
 import React from "react";
+import styles from "./ObjectStatus.module.css";
+
 import { useGlobalContext } from "../../context/GlobalContext";
 import { useEditorContext } from "../../context/EditorContext";
 import ObjectLoader from "./ObjectLoader";
 
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import HelpOutlineIcon from "@mui/icons-material/HelpOutline";
+import ModeStandbyIcon from "@mui/icons-material/ModeStandby";
+
 export interface ObjectStatusProps {
     pid: string;
 }
 
+function getStatusIcon(statusText: string) {
+    switch (statusText) {
+        case "Active":
+            return <CheckCircleIcon></CheckCircleIcon>;
+        case "Inactive":
```

Comment:
I think we need a better icon here for deleted -- the default question mark is weird on deleted items.